Added sample of using VMware Storage Policy API

verdel opened the pull request:
I would like to add an example implementation of:
 - Getting information about VM Home and Virtual Disk storage policies
 - Assignment storage policy to the VM Home and Virtual Disks

using the VMware Storage Policy API.

verdel commented:
Hi, @pgbidkar can you please review the PR?

pgbidkar commented:
> Hi, @pgbidkar can you please review the PR?

Hi @verdel , I am extremely sorry. I am not sure how I missed this PR. Please give me a day time I will review it

Thanks for the patience 
Pavan

verdel commented:
> Hi @verdel , I am extremely sorry. I am not sure how I missed this PR. Please give me a day time I will review it

No problem. I will wait. Thank you for your time.

pgbidkar commented on samples/get_vm_storage_policy.py:
```diff
@@ -0,0 +1,276 @@
+    return obj
+
+
+def GetArgs():
```
Please move this to top of the file to be in consistent with other  sample files

verdel replied:
Changes have also been made in samples/set_vm_storage_policy.py

pgbidkar commented on samples/get_vm_storage_policy.py:
```diff
@@ -0,0 +1,276 @@
+    """
+    parser = argparse.ArgumentParser(
+        description='Show VM Home and Virtual Disk Storage Policies')
+    parser.add_argument('-s', '--host',
```
Use library argument builder

verdel replied:
Changes have also been made in samples/set_vm_storage_policy.py

verdel commented:
I changed the scripts a little bit more. I added the option --set_vm_home in samples/set_vm_storage_policy.py. Setting the storage policy for VM Home is now optional.

pgbidkar commented:
@kumahesh , can you review this sample? Also please verify the sample

pgbidkar commented:
Hi @verdel , Please can you provide your sample test results? Screen capture of command prompt where Sample is run successfully

verdel commented:
> Hi @verdel , Please can you provide your sample test results? Screen capture of command prompt where Sample is run successfully

Sure, no problem.
[![asciicast](https://asciinema.org/a/261278.svg)](https://asciinema.org/a/261278)

kumahesh commented:
Works as expected. It is really valuable contribution. It will be merged soon.

pgbidkar commented:
Hi @verdel , Thanks for the contribution. Merging the PR
